adapters/flipp/flipp.go

```diff
@@ -0,0 +1,199 @@
+package flipp
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/buger/jsonparser"
+	"github.com/gofrs/uuid"
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+var BANNER_TYPE = "banner"
```

Comment:
Can we use `const` for constants?

Reply:
fixed, using `const` for the string types but the arrays still need to use `var`